src/concepts/html/README.md

```diff
@@ -2,15 +2,24 @@
 description: The procedural macro for generating HTML and SVG
 ---
 
-# Using html!
+# Using `html!`
 
 The `html!` macro allows you to write HTML and SVG code declaratively. It is similar to JSX \(a Javascript extension which allows you to write HTML code inside of Javascript\).
 
 **Important notes**
 
 1. The `html!` macro only accepts one root html node \(you can counteract this by [using fragments or iterators](lists.md)\)
 2. An empty `html! {}` invocation is valid and will not render anything
-3. Literals must always be quoted and wrapped in braces: `html! { "Hello, World" }`
+3. Literals in element content must always be quoted and wrapped in braces
+  * `html! { "Hello, World" }`
+  * `html! { <div>{ "Hell, World" }</div> }`
+  * `html! { <div>{ String::from("foo") + "bar" }</div>`
+4. Quoted attribute values are taken literally
+  * `html! { <div> id="bar"</div> }`
+5. Unquoted attribute values are interpreted as expressions
```

Comment:
```suggestion
5. Unquoted attribute values are interpreted as expressions (which must be valid Rust expressions)
```

Reply:
What about just 
’`’
Unquoted attribute values are interpreted as Rust expressions
```
I'd say that they need to be valid does not need to be mentioned.

Reply:
I think it clears any confusion as to whether expressions feature any syntax extensions upon those of regular Rust.